### ISSUE TITLE
Insufficient contrast on card component tags and meta copy

### ISSUE THREAD

**Issue by alexgibson** [mozilla/protocol#165]
The mozorg [homepage](https://www.mozilla.org/) is failing a11y checks on Lighthouse for color contrast on the card tag and meta data copy:

https://www.webpagetest.org/lighthouse.php?test=180703_DH_4ab83e128fbebcb53d7077ceba9b185a

e.g.

```html
<div class="mzp-c-card-tag ">fastcompany.com</div>
<p class="mzp-c-card-meta">May 15, 2018</p>
```

![card](https://user-images.githubusercontent.com/400117/42214466-849dddfc-7eb3-11e8-96ee-b5c06ff08367.png)

We should update this to a gray token color that has better constrast

**Comment by Deepikap1** [comment]
Hey @alexgibson !!
I would like to work on this issue.

**Comment by jenniferharmon** [comment]
Hi @alexgibson! The card tag and card meta elements have a color of #4a4a4f on a white bg. According to contrast checkers, this is a contrast ratio of 8.81:1, which meets accessibility requirements. Would you still like it to be made darker?

**Comment by alexgibson** [comment]
@jenniferharmon it looks like this may have already been fixed, thanks for checkin in here 👍
